css/amp.css

```diff
@@ -422,6 +422,16 @@ i-amphtml-sizer {
 .i-amphtml-element > [placeholder].amp-hidden {
   visibility: hidden;
 }
+/* Placeholder, by design, expands layout=container unlike other layouts where
```

Comment:
I think this means we can remove this:

https://github.com/ampproject/amphtml/blob/3b6024d439b09ee2c4fff1fd49bec6be53329a6c/css/amp.css#L503-L510